ivy/functional/frontends/paddle/creation.py

```diff
@@ -87,6 +87,9 @@ def empty_like(x, dtype=None, name=None):
     return ivy.empty_like(x, dtype=dtype)
 
 
+@with_supported_dtypes(
+    {"2.6.0 and below": ("float32", "float64", "int32", "int64")}, "paddle"
```

Comment:
Why `float16` is not added? ref: https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/eye_en.html

Reply:
Sorry, I missed it. I just added it and tested once again, all the backends are passing.
![image](https://github.com/unifyai/ivy/assets/87087741/671754fa-8712-428c-88b4-13887a3bb3a5)